src/main/java/org/jvnet/hudson/update_center/MavenArtifact.java

```diff
@@ -165,27 +166,26 @@ public String getTimestampAsString() throws IOException {
 
     public Date getTimestampAsDate() throws IOException {
         long lastModified = getTimestamp();
-        SimpleDateFormat bdf = getDateFormat();
-
-        Date tsDate;
         
-        try {
-            tsDate = bdf.parse(bdf.format(new Date(lastModified)));
-        } catch (ParseException pe) {
-            throw new IOException(pe.getMessage());
-        }
 
-        return tsDate;
+        Date lastModifiedDate = new Date(lastModified);
+        Calendar cal = new GregorianCalendar();
+        cal.setTime(lastModifiedDate);
+        cal.set(Calendar.HOUR_OF_DAY, 0);
+        cal.set(Calendar.MINUTE, 0);
+        cal.set(Calendar.SECOND, 0);
+        cal.set(Calendar.MILLISECOND, 0);
+        return cal.getTime();
     }
     
     public static SimpleDateFormat getDateFormat() {
-        return new SimpleDateFormat("MMM dd, yyyy", Locale.US);
+        SimpleDateFormat sdf = new SimpleDateFormat("MMM dd, yyyy", Locale.US);
+        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Comment:
Time stamps appear to be one hour off from what's shown on Artifactory and I think this was intended to fix that, but didn't work out.